composer/utils/checkpoint.py

```diff
@@ -313,7 +313,7 @@ def load_sharded_checkpoint(
             f'Sharded checkpoint loading requires torch version >= 2.0.0. You have torch version {torch.__version__}')
 
     using_multinode = dist.get_world_size() != dist.get_local_world_size()
-    if not using_torch_2_0_1() and using_multinode:
+    if not version.parse(torch.__version__) >= version.parse('2.0.1') and using_multinode:
```

Comment:
should this be 2.1.0?

Reply:
nope, just removing utility fn and only doing for 2.0 to be consistent